### PR TITLE
PR9998-Follow-up 3.10 Release Notes

### PR DESCRIPTION
**Pull request by bmcelvee**
Follow-up to https://github.com/openshift/openshift-docs/pull/9998

### CODE REVIEW

File: release_notes/ocp_3_10_release_notes.adoc [hunk 1]
```diff
@@ -370,6 +370,14 @@ $ oc adm prune groups --sync-config=path/to/sync/config [<options>]
 ----
 See xref:../admin_guide/pruning_resources.adoc#pruning-groups[Pruning groups] for more information.
 
+[[ocp-310-local-ephemeral-storage]]
```
Comment (bmcelvee) [inline release_notes/ocp_3_10_release_notes.adoc]: @tmorriso-rh I added some draft text for this feature, the original PR mentioned adding it to the release notes. PTAL. Thanks!

Comment (tmorriso-rh) [reply release_notes/ocp_3_10_release_notes.adoc]: @bmcelvee This is the note we have in the ephemeral storage docs:

"This topic applies only if the ephemeral storage technology preview is enabled in {product-title} 3.10. This feature is disabled by default. If enabled, the {product-title} cluster uses ephemeral storage to store information that does not need to persist after the cluster is destroyed.  See Configuring Ephemeral Storage for more information."

Maybe include the part about the cluster using ephemeral storage to store information that does not need to persist after the cluster is destroyed? Just for consistency across docs. Thanks!

Comment (bmcelvee) [reply release_notes/ocp_3_10_release_notes.adoc]: Thanks @tmorriso-rh! I added that bit. :stars:

File: release_notes/ocp_3_10_release_notes.adoc [hunk 2]
```diff
@@ -88,7 +88,7 @@ glusterFS.
 . The underlying PV is resized.
 
 [[ocp-310-ceph-provisioner]]
-==== Ceph Provisioner
+==== Ceph Provisioner (Technology Preview)
```
Comment (bmcelvee) [inline release_notes/ocp_3_10_release_notes.adoc]: Need more information on this.

Comment (bmcelvee) [reply release_notes/ocp_3_10_release_notes.adoc]: Need more information on Local Storage too (line 112)

Comment (tmorriso-rh) [reply release_notes/ocp_3_10_release_notes.adoc]: The local storage and ephemeral storage topics are the same.

Comment (tmorriso-rh) [reply release_notes/ocp_3_10_release_notes.adoc]: The Ceph Provisioner feature did not make the 3.10 release. There is more information in the comments: https://bugzilla.redhat.com/show_bug.cgi?id=1568345

Comment (bmcelvee) [reply release_notes/ocp_3_10_release_notes.adoc]: Thank you, @tmorriso-rh! You saved the day :cat: that helps a ton!

File: release_notes/ocp_3_10_release_notes.adoc [hunk 3]
```diff
@@ -577,6 +621,29 @@ CLI] for information on how to install and write extensions for the CLI.
...
+
+[[ocp-310-security-specify-whitelist-cipher-suite-for-etcd]]
+==== Specify whitelist Cipher Suite for etcd
+
+You can set whitelist cipher suites for use with etcd in order to meet security
+policies.
+
+For more information, see xref:../install_config/master_node_configuration.adoc#master-config-tls-cipher[Specifying TLS ciphers for etcd]
+
+[[ocp-310-control-sharing-pid-namespace-between-containers]]
```
Comment (bmcelvee) [inline release_notes/ocp_3_10_release_notes.adoc]: This is a placeholder and may need to be removed.

File: release_notes/ocp_3_10_release_notes.adoc [hunk 4]
```diff
@@ -577,6 +621,29 @@ CLI] for information on how to install and write extensions for the CLI.
...
+==== Control Sharing the PID Namespace Between Containers
+
+WIP
+
+[[ocp-310-router-service-account-access-secrets]]
```
Comment (bmcelvee) [inline release_notes/ocp_3_10_release_notes.adoc]: This is a placeholder and may need to be removed.

File: release_notes/ocp_3_10_release_notes.adoc [hunk 5]
```diff
@@ -577,6 +621,29 @@ CLI] for information on how to install and write extensions for the CLI.
 * The *_xterm.js_* dependency for pod terminal was updated with greatly improved performance.
 * You can now create image pull secrets directly from the deploy image dialog.
 
+[[ocp-310-security]]
+=== Security
+
+[[ocp-310-security-specify-whitelist-cipher-suite-for-etcd]]
```
Comment (bmcelvee) [inline release_notes/ocp_3_10_release_notes.adoc]: The PR for this one is and being worked on, so the link is not yet active.

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL :unicorn:

**Review by kalexand-rh — COMMENTED** [review]
Couple of picks, but overall LGTM.

**Comment by bmcelvee** [comment]
I am going ahead and merging this PR since it has been peer reviewed and there will be more follow-ups to https://github.com/openshift/openshift-docs/pull/9998. A larger review will follow.

**Pull request merged**